src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -6,7 +6,9 @@
 using System.Linq;
 using System.Reflection;
 using System.Text.RegularExpressions;
+using Microsoft.AspNet.Razor.Parser;
 using Microsoft.AspNet.Razor.TagHelpers;
+using Microsoft.AspNet.Razor.Text;
```

Comment:
ugly that these additional namespaces are required.

please file a follow-up issue to rename `ParserErrorSink` and move it to somewhere more-accessible.  that class has been used for more than just parser errors for a while.  should probably also move `RazorError` (currently buried deep in the SyntaxTree namespace) and `SourceLocation`.